cylc/flow/task_events_mgr.py

```diff
@@ -591,6 +592,16 @@ def process_message(
                 itask, severity, message, event_time, flag, submit_num):
             return None
 
+        severity = cast(int, LOG_LEVELS.get(severity, INFO))
```

Comment:
Moved out of `_process_message_check` because they only run if it is true.

Reply:
It's not meant to change anything.

Reply:
What did you mean by this then:

> Moved out of `_process_message_check` because they only run if it is true.

Reply:
It only runs in `process_message_check` if that method is going to return true. Here it only runs after process message check returns true and not otherwise. I thought it was more helpful to understanding the flow of the program here.

Reply:
Turns out you were right - I shouldn't have moved this. I was wrong to thing that `event_time` in process_event == `timestamp` in process message check and a couple of tests were broken by this. Please hit resolve if you are satisfied.